flake.nix

```diff
@@ -510,6 +511,7 @@
               # sydtest dependencies
               ps.sydtest
               ps.sydtest-discover
+              ps.sydtest-hedgehog
```

Comment:
We're actually considering using quickcheck instead, which wouldn't need this, would you mind fixing that?

Reply:
it didn't even use the dependency anywher